packages/EasyCore/src/Bridge/Symfony/ApiPlatform/Normalizers/CarbonNormalizer.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Bridge\Symfony\ApiPlatform\Normalizers;
+
+use Carbon\CarbonImmutable;
+use Carbon\CarbonInterface;
+use Carbon\CarbonTimeZone;
+use Symfony\Component\Serializer\Exception\InvalidArgumentException;
+use Symfony\Component\Serializer\Normalizer\CacheableSupportsMethodInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+final class CarbonNormalizer implements NormalizerInterface, CacheableSupportsMethodInterface
+{
+    /**
+     * @var string
+     */
+    private const FORMAT_KEY = 'datetime_format';
+
+    /**
+     * @var string
+     */
+    private const TIMEZONE_KEY = 'datetime_timezone';
+
+    /**
+     * @var mixed[]
+     */
+    private $defaultContext = [
+        self::FORMAT_KEY => CarbonImmutable::RFC3339,
```

Comment:
Let's consider the following refactoring:

1. Seems we can simply introduce consts with default values instead of this array
2. We have to make these default values configurable to be able to pass own default format and own default timezone

Reply:
done